public/themes/modern.css

```diff
@@ -597,3 +597,14 @@ div.id4 {
   align-self: flex-start;
   padding: 4px 4px 0 0;
 }
+
+/** handle the popup color and the text color based on the theme **/
+.swal2-popup {
+  background: #3c3f44;
+  color: #e1e1e1;
+}
+
+body.swal2-shown > [aria-hidden="true"] {
+  transition: 0.1s filter;
+  filter: blur(3px);
```

Comment:
`filter: blur` looks out of place compared to the other overlays in the frontend who just darken the background content (`opacity: 0.6`); I don't think this should be added.